README.md

```diff
@@ -159,6 +159,8 @@ Anyone is invited to contribute to the project in-line with the [contribution gu
 
 Example of a Grafana dashboard linked to OpenFaaS showing auto-scaling live in action:
```

Comment:
I'm seeing " Example of a Grafana dashboard linked to OpenFaaS showing auto-scaling live in action:" duplicated. Please can you link to "Original" dashboard here too.